BondageClub/Scripts/Reputation.js

```diff
@@ -41,9 +41,10 @@ function ReputationChange(RepType, RepValue, Push) {
 	}
 
 	else {
-		throw new Error("trying to change invalid reputation")
+		if (CurrentScreen != "Login") {
+			throw new Error("trying to change invalid reputation")
```

Comment:
```suggestion
			console.warn(`Invalid reputation type "${RepType}"`);
```
I didn't have a proper look at the previous PR before it was merged, so didn't notice this then, but I feel like throwing errors here is perhaps a little harsh. I don't feel that using an invalid reputation type should be game-breaking, but we should definitely be warning about it. 

I'd suggest maybe switching them to use a `console.warn`. Including the invalid type in the message might also be useful information for future devs, particularly if they want to add new reputation/skill types.